sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_input_outputs.py

```diff
@@ -26,6 +43,8 @@ class InternalInput(Input):
     def __init__(self, *, datastore_mode=None, is_resource=None, **kwargs):
         self.datastore_mode = datastore_mode
         self.is_resource = is_resource
+        if "type" in kwargs:
+            kwargs["type"] = self._convert_ui_widget_type(kwargs["type"])
```

Comment:
Could you point me to the component (spec) which cause this issue on teams?

Reply:
Offline sync. There is no component spec because it occurs on a pipeline component